### ISSUE TITLE
Add support for Property Profiles

### ISSUE THREAD

**Issue by philsbln** [ietf-tapswg/api-drafts#325]
This issue was discussed in the May 2019 interim.

After fist implementation experiences, we feel that we should add a convenient way to express common Transport Parameter combinations as some kind of profile, e.g. "TCP like" meaning "Reliable, in-order, non-message preserving"

**Comment by philsbln** [comment]
Proposal how to solve this:

Replace the TransportProperties constructor in section 5.2 with
~~~
   TransportProperties := NewTransportProperties(propertyProfile?)
~~~

@gorryfair promised this at the interim. I only open this issue to document the proposal how to solve this.

**Comment by MaxF12** [comment]
This can be closed.

**Comment by philsbln** [comment]
Closed by #328
